dev/com.ibm.ws.security.javaeesec/resources/com/ibm/ws/security/javaeesec/internal/resources/JavaEESecMessages.nlsprops

```diff
@@ -67,3 +67,31 @@ JAVAEESEC_WARNING_NO_PWD.useraction=If a password was expected, review the calle
 JAVAEESEC_WARNING_MULTI_CALLER=CWWKS1924W: The ''{1}'' query returned multiple results for the {0} caller on the DatabaseIdentityStore.
 JAVAEESEC_WARNING_MULTI_CALLER.explanation=Multiple results were returned for the requested caller. The caller query should only return a single result.
 JAVAEESEC_WARNING_MULTI_CALLER.useraction=Review the caller query and database contents. Change the caller query to return one result.
+
+JAVAEESEC_ERROR_EXCEPTION_ON_BIND=CWWKS3400W: Binding with the {0} caller has failed. The following exception occurred: {1}
+JAVAEESEC_ERROR_EXCEPTION_ON_BIND.explanation=An exception occurred while binding with the provided caller.
+JAVAEESEC_ERROR_EXCEPTION_ON_BIND.useraction=Review the provided exception.
+
+JAVAEESEC_ERROR_EXCEPTION_ON_SEARCH=CWWKS3401E: Searching for the {0} caller with the {1} filter and the {2} searchBase failed. The following exception occurred: {3}
+JAVAEESEC_ERROR_EXCEPTION_ON_SEARCH.explanation=An exception occurred while searching for the provided caller.
+JAVAEESEC_ERROR_EXCEPTION_ON_SEARCH.useraction=Review the provided exception.
+
+JAVAEESEC_ERROR_EXCEPTION_ON_GROUP_SEARCH=CWWKS3402E: Searching for groups of {0} has failed. The following exception occurred: {1}
```

Comment:
Change to:
Searching for groups of {0} failed. The following exception occurred: {1}